differ/differ.go

```diff
@@ -143,14 +143,16 @@ func moduleToRuleName(module string) string {
 	return result
 }
 
-func findRuleByNameAndErrorKey(ruleContent types.RulesMap, impacts types.Impacts, ruleName string, errorKey string) (int, int, int) {
+func findRuleByNameAndErrorKey(ruleContent types.RulesMap, impacts types.Impacts, ruleName string, errorKey string) (int, int, int, string) {
```

Comment:
for the future it might be better to use named return values

Reply:
True, true.